src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -8,14 +8,21 @@
 @import '../../styles/shared-styles.scss';
 @import '../../../node_modules/office-ui-fabric-core/dist/sass/References';
 
-$font-size: var(--font-size, #{$ms-font-size-m});
-$font-weight: var(--font-weight, #{$ms-font-weight-semibold});
-$color: var(--color, #{$ms-color-neutralDark});
+$person-card-display-name-font-size: var(--person-card-display-name-font-size, 27.3103px);
+$person-card-secondary-title-font-size: var(--person-card-secondary-title-font-size, 18.2069px);
```

Comment:
This can be simplified to `--person-card-title-*`

Reply:
final here:

dc91e65